static/style.css

```diff
@@ -240,6 +240,96 @@ marquee{
             border-radius: 5px;
             background-color:rgb(67, 67, 163);
         }
+
+/* Baby and kids section */
+
+.carousel{
+  width:100%;
+  height:500px;
+  /* left: 400px; */
+
+}
+
+ .carousel-item img{
+  height: 500px;
+  width: 300px;
+   
+ }
+ 
+p{
+  padding: 20px 50px 20px 50px;
+}
+#products{
+  padding: 20px 40px 30px 40px;
+/*   margin: 0 40px 0 40px; */
+  
+  
+}
+
+.image{
+  padding: 30px 30px 50px 30px;
+  margin-left: 20px;
+  margin-right: 20px;
+  position: relative;
+  display: inline-block;
+  box-shadow: 1rem 1rem 1rem -0.9rem rgb(81, 3, 145);
+  border-radius: 30px;
+
+  /* margin: auto;
+  text-align: center; */
+}
+h3{
+  margin-top: 40px;
+  color: rgb(6, 57, 99);
+  
+  
+}
```

Comment:
change all tag you have added that is not class or id 
and add class or which is parent of it like

```
h3{
  margin-top: 40px;
  color: rgb(6, 57, 99);


}
```


change this to 


```
.offer h3{
  margin-top: 40px;
  color: rgb(6, 57, 99);


}
```